# -------------------------------------------------------------------
# Some SCEP servers (e.g. openxpki) are incorrectly doing certificate
# renewal via messageType PKCSReq (19) instead of RenewalReq (17).
# -------------------------------------------------------------------
--- a/conf/options/pki.conf
+++ b/conf/options/pki.conf
@@ -1,21 +1,6 @@
 pki {
-
-    # Plugins to load in the pki tool.
-    # load =
-
     scep {
-
-        # Source IP address to bind for HTTP operations.
-        # http_bind =
-
-        # Timeout for HTTP operations.
-        # http_timeout = 30s
-
-        # Some SCEP servers (e.g. openxpki) are incorrectly doing certificate
-        # renewal via messageType PKCSReq (19) instead of RenewalReq (17).
-        # renewal_via_pkcs_req = no
-
+        renewal_via_pkcs_req = yes
     }
-
 }
 
